.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -33,7 +33,14 @@ body:
       label: Logs
       description: Add any logs/error messages to help debug the issue.
     validations:
-      required: true
+      required: false
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Upload any screenshots to help debug the issue.
```

Comment:
@dhawal89 This is for external users who don't have access to our task management tool.

Reply:
@dhawal89 as @danfowler mentioned, this is for community contributions. Including screenshots in our issue template helps make things clearer and easier to communicate. It will help users show exactly what's going wrong in a way that words might miss.

Reply:
Thanks @danfowler, @aashishgurung and @dhawal89 

Merging